cli.py

```diff
@@ -165,11 +168,10 @@ def train(dataroot, name, gpu_ids, checkpoints_dir, targets_no, input_nc, output
     plot, and save models.The script supports continue/resume training.
     Use '--continue_train' to resume your previous training.
     """
+    local_rank = os.getenv('LOCAL_RANK') # DDP single node training triggered by torchrun has LOCAL_RANK
+    rank = os.getenv('RANK') # if using DDP with multiple nodes, please provide global rank in env var RANK
 
     if len(gpu_ids) > 0:
-        local_rank = os.getenv('LOCAL_RANK') # DDP single node training triggered by torchrun has LOCAL_RANK
```

Comment:
bug fix:
this needs to be moved out of the if clause, so local_rank and rank are always initialized (even in no-gpu situations) and can be checked later (line 180 in this file).